From: handsome_feng <jianfengli@ubuntukylin.com>
Date: Sat, 16 Jul 2022 14:27:46 +0800
Subject: update changelog

---
 src/core/system.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/system.conf.in b/src/core/system.conf.in
index 6d05a3c..06b9d79 100644
--- a/src/core/system.conf.in
+++ b/src/core/system.conf.in
@@ -39,7 +39,7 @@
 #DefaultStandardOutput=journal
 #DefaultStandardError=inherit
 #DefaultTimeoutStartSec=90s
-#DefaultTimeoutStopSec=90s
+DefaultTimeoutStopSec=10s
 #DefaultTimeoutAbortSec=
 #DefaultRestartSec=100ms
 #DefaultStartLimitIntervalSec=10s
